wonnx/templates/pool/aggregate.wgsl

```diff
@@ -56,5 +75,66 @@ fn main([[builtin(global_invocation_id)]] global_id: vec3<u32>) {
 			let index = base_index_2 + index_vec * {{ o_chunks[0][1] }}u;
 			output_0.data[index] = result[index_vec];
 		}
+	} else {
+	if ((gidx >= 4u * {{ o_lens[0] / 4 }}u) && ( gidx < {{ o_lens[0] }}u)) {
```

Comment:
I don't get it, doesn't this just mean `gidx >= o_lens[0] && gidx < o_lens[0]` which is still not possible? @haixuanTao

Reply:
Well it is an euclidean division. So it is between `[ o_lens - o_lens % 4, o_lens ]`